eng/Npm.Workspace.FunctionalTests.nodeproj

```diff
@@ -1,4 +1,4 @@
-<Project>
+<Project DefaultTargets="Build">
```

Comment:
What made this necessary?

Reply:
This is just correctness as without a DefaultTargets, the very first target gets built which is the Restore target in this project file. Not necessary for this to be green but the right thing to do.